detox/README.md

```diff
@@ -50,26 +13,16 @@ Debugging(from the root directory of the particular native-widget): `npm run deb
 
 PLATFORM_NAME = `android` or `ios`
 
-# General improvements
-- List all versions of the working tools 
-    - Android studio
```

Comment:
Devs still need Android Studio to get e.g. sdkmanager, right? Should we keep this as a dependency?

Reply:
Added a list of required software.